doc/operations/code_upgrade.rst

```diff
@@ -0,0 +1,191 @@
+Code Upgrade
+============
+
+.. note:: Refer to :doc:`/operations/code_upgrade_1x` for specific guidelines on how to upgrade a 1.x CCF service to 2.0.
+
+This page describes how operators/members can upgrade a live CCF service to a new version with minimal downtime.
```

Comment:
I'm not sure how to correctly convey the "minimal downtime" part. Is it really "no downtime at all" since the service is always available for read-only transactions during the code upgrade procedure, but write transactions cannot be processed while the election is happening (which can also happen in a normal election process)? I'm leaning towards "no downtime at all".

Reply:
I think "minimal downtime" is fair, there is some write downtime due to the inevitable election, but it's no worse than what can normally happen outside an upgrade if f=1.